TypeError: fs.existsSync is not a function

When I follow the process in README, no page is shown in the browser. The firefox dev tools report
"TypeError: fs.existsSync is not a function"

Steps:
gulp dev
[09:04:41] Using gulpfile ~/git/loklak_webclient/gulpfile.js
[09:04:41] Starting 'clean'...
[09:04:41] Finished 'clean' after 17 ms
[09:04:41] Starting 'dev'...
[09:04:41] Starting 'styles'...
[09:04:41] Starting 'images'...
[09:04:41] Starting 'fonts'...
[09:04:41] Starting 'views'...
[09:04:41] Starting 'browserify'...
[09:04:42] Rebundle...
[09:04:42] Finished 'images' after 93 ms
[09:04:42] Finished 'views' after 162 ms
[09:04:44] Finished 'styles' after 2.07 s
[09:04:44] Finished 'fonts' after 2.04 s
[09:04:47] Finished 'browserify' after 5.15 s
[09:04:47] Starting 'browserSync'...
[09:04:47] Finished 'browserSync' after 58 ms
[09:04:47] Starting 'server'...
[09:04:47] Finished 'server' after 11 ms
[09:04:47] Starting 'watch'...
[09:04:47] Finished 'watch' after 51 ms
[09:04:47] Finished 'dev' after 5.31 s
[BS] Proxying: http://localhost:3000
[BS] Now you can access your site through the following addresses:
[BS] Local URL: http://localhost:3001
[BS] External URL: http://192.168.1.56:3001
GET / 200 13.888 ms - 462
GET / 200 6.120 ms - 462
GET /css/main.css 200 11.439 ms - 276914
GET /js/main.js 200 333.890 ms - 3262523

Then the browser opens http://localhost:3001/
it shortly displays "Connected to Browser Sync" and then shows only an empty page
- I open the Firefox JS console and reload
- the console prints out the error "TypeError: fs.existsSync is not a function" in  main.js:35192:8

this is inside node.js. I tested my node version:
$ node --version
v0.12.2

This bug may occur to me because I am a node.js newbie and forget to do something. But then that 'something' should appear in the readme.

Reinstalling the code all fresh
```sh
npm ERR! notarget Valid install targets:
npm ERR! notarget ["0.1.2","0.1.3","0.2.0","0.3.0","0.4.0","0.4.1","0.4.2","0.4.3","0.4.4","0.4.6","0.4.8","0.5.0","0.6.0","0.6.1","1.0.0","1.5.0","1.5.1","1.5.2","1.5.3","1.6.0","1.6.1","1.6.2","1.7.0","1.8.0","1.8.1"]
npm ERR! notarget
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! System Darwin 14.0.0
npm ERR! command "node" "/usr/local/bin/npm" "install"
npm ERR! cwd /Users/sudheeshsinganamalla/Documents/loklak_GSoC/webclient2
npm ERR! node -v v0.10.26
npm ERR! npm -v 1.4.3
npm ERR! code ETARGET
npm http 304 https://r
```
Able to reproduce the problem. Should be something about the new library. Fixes on its way

--- a/app/js/constants.js
+++ b/app/js/constants.js
@@ -1,6 +1,5 @@
 'use strict';
-var env = require('node-env-file');
-env('../../.env');
+require('dotenv').config({path: '.env'});
 
 var AppSettings = {
   apiUrl: process.env.apiUrl

--- a/package.json
+++ b/package.json
@@ -73,6 +73,7 @@
   "dependencies": {
     "angular-loading-bar": "^0.7.1",
     "angular-moment": "^0.10.0",
-    "node-env-file": "^0.1.7"
+    "node-env-file": "^0.1.7",
+    "dotenv": "^1.1.0"
   }
 }

